python/lsst/meas/base/plugins.py

```diff
@@ -198,10 +201,21 @@ def measure(self, measRecord, exposure):
         # Compute the median variance value for each pixel not excluded by the mask and write the record.
         # Numpy median is used here instead of afw.math makeStatistics because of an issue with data types
         # being passed into the C++ layer (DM-2379).
-        medVar = numpy.median(pixels.getVarianceArray()[logicalMask])
-        measRecord.set(self.varValue, medVar)
+        if numpy.any(logicalMask):
+            medVar = numpy.median(pixels.getVarianceArray()[logicalMask])
+            measRecord.set(self.varValue, medVar)
+        else:
+            raise bl.MeasurementError("Footprint empty, or all pixels are masked, can't compute median",
```

Comment:
Punctuation is a bit funky. I'd drop the first comma altogether and turn the second into a colon.